travis/main.sh

```diff
@@ -104,7 +104,6 @@ build_cmake() {
 		-DENABLE_PNG=1
 		-DENABLE_TIFF=1
 		-DENABLE_WEBP=1
-		-DENABLE_XPM=1
```

Comment:
`libxpm-dev` is unavaible in brew repo, while avaible in apt repo. But I should not delete `-DENABLE_XPM=1` directly. A better way should be enabling different feature according to different OS.

Reply:
Besides `libxpm-dev`, `libraqm-dev` faces the same problem.